My link: rephrase (#520)
hacs/documentation: source/docs/use/my.md

---
title: 'My Home Assistant'
description: 'Creating a My link to a HACS repository'
---

> **Home Assistant My links** can be used to link directly to an integration inside HACS. This is used for example, to share a link on Discord or to bookmark it. My links work for both default and custom repositories.

Reviewer: ```suggestion
**Home Assistant My links** can be used to link directly to a repository inside HACS. This is used for example, to share a link on Discord or to bookmark it. My links work for both default and custom repositories.
```

## To create a My link using a hotkey

1. In Home Assistant, open the repository of interest and press the `m` key.
      - The My link page opens, with the information already filled in. 
2. Copy the URL or snippets you want for your My link.

## To create a My link manually

1. Go to https://my.home-assistant.io/create-link/?redirect=hacs_repository.
2. Fill in the fields on that page to generate your My link.

>    - **Owner**: the author of the repository. In the example of spook, this is "frenck"

Reviewer: ```suggestion
   - **Owner**: the author of the repository. In the example of spook (`frenck/spook`), this is "frenck"
```

   - **Repository**: the name of the repositroy. For example: "spook"

>    - **Category** is only needed for custom repositories. For example: "integration".

Reviewer: I have no good suggestion here, the field is named "category" on that site, so that needs to stay.
But this "category" is what we refer to as "type" in HACS going forward.

>    - **Category**. efers to the type of repository. For example: "integration". Only needed for custom repositories.

Reviewer: ```suggestion
   - **Category**. refers to the type of repository. For example: "integration". Only needed for custom repositories.
```

---

> id: my

Reviewer: ```suggestion
```
